nf/models.py

```diff
@@ -30,5 +29,5 @@ def inverse(self, z):
     def sample(self, n_samples):
         z = self.prior.sample((n_samples,))
         x, _ = self.inverse(z)
-        return x
+        return x.to('cpu')
```

Comment:
Don't think sending to CPU here is necessary. We can let the samples stay on the same device as `self.prior`.